services/web/client/source/class/osparc/utils/Study.js

```diff
@@ -41,7 +41,9 @@ qx.Class.define("osparc.utils.Study", {
               const newUuid = osparc.utils.Utils.uuidv4();
               const minStudyData = osparc.data.model.Study.createMyNewStudyObject();
               minStudyData["name"] = service["name"];
-              minStudyData["thumbnail"] = service["thumbnail"];
+              if (service["thumbnail"]) {
+                minStudyData["thumbnail"] = service["thumbnail"];
```

Comment:
The data in the enveloped body for ``GET /catalog/services/{service_key}/{service_version}``
has the following schema (notice the red * for required fields)
![Screen Shot 2021-09-22 at 13 35 48](https://user-images.githubusercontent.com/32402063/134336588-7ff5860e-8c72-4ca3-8cff-b75a8c769ca4.png)

```json
"ServiceOut": {
        "title": "ServiceOut",
        "required": [
          "name",
          "description",
          "key",
          "version",
          "type",
          "authors",
          "contact",
          "inputs",
          "outputs"
        ],
        "type": "object",
        "properties": {
          "name": {
            "title": "Name",
            "type": "string",
            "description": "short, human readable name for the node",
            "example": "Fast Counter"
          },
          "thumbnail": {
            "title": "Thumbnail",
            "maxLength": 2083,
            "minLength": 1,
            "type": "string",
            "description": "url to the thumbnail",
            "format": "uri",
            "examples": [
              "https://user-images.githubusercontent.com/32800795/61083844-ff48fb00-a42c-11e9-8e63-fa2d709c8baf.png"
            ]
          },
          "description": {
            "title": "Description",
            "type": "string",
            "description": "human readable description of the purpose of the node",
            "examples": [
              "Our best node type",
              "The mother of all nodes, makes your numbers shine!"
            ]
          },
          "classifiers": {
            "title": "Classifiers",
            "type": "array",
            "items": {
              "type": "string"
            }
          },
          "quality": {
            "title": "Quality",
            "type": "object",
            "default": {}
          },
          "access_rights": {
            "title": "Access Rights",
            "type": "object",
            "additionalProperties": {
              "$ref": "#/components/schemas/ServiceGroupAccessRights"
            },
            "description": "service access rights per group id"
          },
          "key": {
            "title": "Key",
            "pattern": "^(simcore)/(services)/(comp|dynamic|frontend)(/[\\w/-]+)+$",
            "type": "string",
            "description": "distinctive name for the node based on the docker registry path",
            "examples": [
              "simcore/services/comp/itis/sleeper",
              "simcore/services/dynamic/3dviewer"
            ]
          },
          "version": {
            "title": "Version",
            "pattern": "^(0|[1-9]\\d*)(\\.(0|[1-9]\\d*)){2}(-(0|[1-9]\\d*|\\d*[-a-zA-Z][-\\da-zA-Z]*)(\\.(0|[1-9]\\d*|\\d*[-a-zA-Z][-\\da-zA-Z]*))*)?(\\+[-\\da-zA-Z]+(\\.[-\\da-zA-Z-]+)*)?$",
            "type": "string",
            "description": "service version number",
            "examples": [
              "1.0.0",
              "0.0.1"
            ]
          },
          "integration-version": {
            "title": "Integration-Version",
            "pattern": "^(0|[1-9]\\d*)(\\.(0|[1-9]\\d*)){2}(-(0|[1-9]\\d*|\\d*[-a-zA-Z][-\\da-zA-Z]*)(\\.(0|[1-9]\\d*|\\d*[-a-zA-Z][-\\da-zA-Z]*))*)?(\\+[-\\da-zA-Z]+(\\.[-\\da-zA-Z-]+)*)?$",
            "type": "string",
            "description": "integration version number",
            "examples": [
              "1.0.0"
            ]
          },
          "type": {
            "allOf": [
              {
                "$ref": "#/components/schemas/ServiceType"
              }
            ],
            "description": "service type",
            "examples": [
              "computational"
            ]
          },
          "badges": {
            "title": "Badges",
            "type": "array",
            "items": {
              "$ref": "#/components/schemas/Badge"
            }
          },
          "authors": {
            "title": "Authors",
            "minItems": 1,
            "type": "array",
            "items": {
              "$ref": "#/components/schemas/Author"
            }
          },
          "contact": {
            "title": "Contact",
            "type": "string",
            "description": "email to correspond to the authors about the node",
            "format": "email",
            "examples": [
              "lab@net.flix"
            ]
          },
          "inputs": {
            "title": "Inputs",
            "type": "object",
            "description": "definition of the inputs of this node",
            "patternProperties": {
              "^[-_a-zA-Z0-9]+$": {
                "$ref": "#/components/schemas/ServiceInput"
              }
            }
          },
          "outputs": {
            "title": "Outputs",
            "type": "object",
            "description": "definition of the outputs of this node",
            "patternProperties": {
              "^[-_a-zA-Z0-9]+$": {
                "$ref": "#/components/schemas/ServiceOutput"
              }
            }
          },
          "owner": {
            "title": "Owner",
            "type": "string",
            "format": "email"
          }
        },
        "description": "Static metadata for a service injected in the image labels",
        "example": {
          "name": "File Picker",
          "description": "File Picker",
          "classifiers": [],
          "quality": {},
          "access_rights": {
            "1": {
              "execute_access": true,
              "write_access": false
            },
            "4": {
              "execute_access": true,
              "write_access": true
            }
          },
          "key": "simcore/services/frontend/file-picker",
          "version": "1.0.0",
          "type": "dynamic",
          "authors": [
            {
              "name": "Odei Maiz",
              "email": "maiz@itis.swiss"
            }
          ],
          "contact": "maiz@itis.swiss",
          "inputs": {},
          "outputs": {
            "outFile": {
              "displayOrder": 0,
              "label": "File",
              "description": "Chosen File",
              "type": "data:*/*"
            }
          },
          "owner": "maiz@itis.swiss"
        }
      },
```